hrui/tests/phpunit/CRM/HRUI/HelperTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_HRUI_HelperTest
+ *
+ * @group headless
+ */
+class CRM_HRUI_HelperTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  use HRUITrait;
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->installMe(__DIR__)
+      ->install('org.civicrm.hrcase')
```

Comment:
hrcase create some relationship types such as  ( Line manager is ) which is required to test (getLineManagersList) method ... I did that instead of creating the relationship manually to make the tests more realistic .